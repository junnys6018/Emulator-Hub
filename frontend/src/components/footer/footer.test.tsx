import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './footer';

test('<Footer /> snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
});
