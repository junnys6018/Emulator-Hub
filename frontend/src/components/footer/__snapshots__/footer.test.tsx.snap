// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> snapshot 1`] = `
<div
  className="lg:h-32 flex-shrink-0"
  style={
    Object {
      "backgroundImage": "url(test-file-stub)",
      "backgroundSize": "64px",
      "imageRendering": "pixelated",
    }
  }
>
  <div
    className="container h-full flex lg:items-center flex-col lg:flex-row"
  >
    <span
      className="font-bold text-xl my-12 mr-20"
    >
      Emulator Hub
    </span>
    <a
      className="text-sm mb-7 lg:mb-0 mr-12"
      href="https://github.com/junnys6018/Emulator-Hub"
      rel="noopener noreferrer"
      target="_blank"
    >
      Source Code
    </a>
    <a
      className="text-sm mb-7 lg:mb-0 mr-auto"
    >
      Contact
    </a>
    <span
      className="text-sm mb-12 lg:mb-0"
    >
      Designed & Built by Jun Lim
    </span>
  </div>
</div>
`;
